testdata/p4tc_samples_outputs/default_action_with_param_01.template

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/default_action_with_param_01 pipeid 1 numtables 2
+
+$TC p4template create action/default_action_with_param_01/MainControlImpl/next_hop actid 1
```

Comment:
Just to confirm, in this case the action should have a runtime parameter, as per discussion with Jamal (https://github.com/Sosutha/p4c/pull/1#issuecomment-1961376802):
```
$TC p4template create action/default_action_with_param_01/MainControlImpl/next_hop actid 1 param
```

However this will be done in a future PR, right?
As will the creation of a new field to `struct p4tc_filter_fields` to represent this parameter.

Reply:
Yes. It will be a future PR